identityClient/api.go

```diff
@@ -196,6 +198,7 @@ type CompleteUserDeviceRegisterRequest struct {
 	DeviceID        string `json:"device_id"`         // Unique device identifier
 	DeviceName      string `json:"device_name"`       // Human readable device name
 	DevicePublicKey string `json:"device_public_key"` // PublicKey to permanently stored PrivateKey on device
+	TOTP            string `json:"secret"`
```

Comment:
This key is not the actual secret, right? We're sending the 6-digit TOTP?

Reply:
Right this is misleading naming that should be corrected, it's the 6 digit totp for verification.